client/data/timeline/hooks.js

```diff
@@ -27,7 +27,7 @@ export const useTimeline = ( chargeId ) =>
 				};
 			}
 
-			const { payment_intent: intentionId } = getCharge( chargeId );
+			const { payment_intent: intentionId } = getCharge( chargeId ) || {};
```

Comment:
This should probably be `getCharge( chargeId ) ?? {};`

Reply:
I think the difference is that `||` will fallback in case of any false-y values, and while I don't think that's ever likely to happen in this case, I'd consider it better form to use the `??` operator here, since that will only fallback in the case of `null` or `undefined` 🙂

Reply:
Bottom line being; yes, I agree that it probably doesn't matter here, but as a matter of good practice I think we're better off using `??` here ☺️

Reply:
Now using `??` as of 6d0d5eb891b94562536432b627be330e113066d6